lib/generators/samvera/nesting_indexer/templates/samvera-nesting_indexer_initializer.rb

```diff
@@ -0,0 +1,23 @@
+# rubocop:disable Style/FileName
+require 'samvera/nesting_indexer'
+# rubocop:enable Style/FileName
+
+Samvera::NestingIndexer.configure do |config|
+  # How many layers of nesting are allowed for collections
+  # For maximum_nesting_depth of 3 the following will raise an exception
+  # C1 ={ C2 ={ C3 ={ W1
```

Comment:
Is W1 also a collection?  or is it a work?  If max depth is 3, the exception raises when you try to add a 4th, right?  The use of W could be confusing.

Reply:
There is notation guidance in the README, but I will clarify in an amendment.